src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(a) {
+    return function(b) {
+      count++;
+
+      if (count <= 3 || count % 2 === 1) {
+        return a + b;
+      } else {
+        return 'Bzzz... Error!';
+      }
```

Comment:
When you are returning something from if statement, you don't need else afterwards
```suggestion
      if (count <= 3 || count % 2 === 1) {
        return a + b;
      }
       
      return 'Bzzz... Error!';
```